### PR TITLE
[ADD] Module 'oca_psc_team' to store PSC details in project.

### PR DESCRIPTION
**Pull request by st-aprajapati**
This module allows you to create PSC categories and configure PSC details in the project.

### CODE REVIEW

File: oca_psc_team/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
```
Comment (elicoidal) [inline oca_psc_team/README.rst]: please use README fragments

File: oca_psc_team/README.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+.. contents::
+   :local:
+
+Usage
+=====
+#. Go to 'Project' menu > Configuration > 'PSC Categories' menu and create PSC categories.
+#. Go to 'Project' menu > Configuration > Projects and follow below steps:
+
+  #. Go to 'PSC Details' tab and fill details.
```
Comment (elicoidal) [inline oca_psc_team/README.rst]: indent one more space

**Review by elicoidal — APPROVED** [review]
LGTM

**Comment by sbidoul** [comment]
Looks good. Just change to use the readme fragments, and I'll deploy this.

**Comment by st-aprajapati** [comment]
> Looks good. Just change to use the readme fragments, and I'll deploy this.

Sure. I will do it.

File: oca_psc_team/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    'version': '11.0.1.0.0',
+    'category': 'Website',
+    'license': 'AGPL-3',
+    'website': 'https://github.com/OCA/oca-custom',
+    'author': 'Odoo Community Association (OCA), Surekha Technologies',
+    'depends': [
+        'project',
+        'github_connector',
+        'project_members',
+        'website',
```
Comment (sbidoul) [inline oca_psc_team/__manifest__.py]: I think the dependency on website can be removed here.

Comment (st-aprajapati) [reply oca_psc_team/__manifest__.py]: Yes, we can remove this.

**Comment by sbidoul** [comment]
@st-aprajapati could you remove the last commit with the website module? So I can merge this one and you can another PR with the website part. It will be easier.

**Comment by st-aprajapati** [comment]
> @st-aprajapati could you remove the last commit with the website module? So I can merge this one and you can another PR with the website part. It will be easier.

Commit reverted.

**Pull request merged**